### ISSUE TITLE
Custom function arguments and Learn more about the custom function are missing in Chrome

### ISSUE THREAD

**Issue by leybal** [OfficeDev/office-js#3228]
Hi
Our team and I working on Excel add-in and we noticed that latest UI updates have affected tooltips for Excel online functions. It looks like this:

![Screenshot 2023-03-17 150345](https://user-images.githubusercontent.com/9270295/225913137-493b00a1-fca2-4747-9dcb-29bd0f5fed5c.png)

The standard Excel function has tooltips regarding formula arguments. There is also the Learn more about function link.
Previously there were tooltips for custom formulas. There are no hints right now. It looks like this:

![custom_func](https://user-images.githubusercontent.com/9270295/225915731-31bc015c-59c3-47bc-9438-232dfba155e9.png)

Although in the functions.js file, the arguments have a description and help Url.
Is this a bug? Or updates are planned for custom functions UI. It would be good to know your plans for custom functions UI and deadlines. Thank you.

**Comment by leybal** [comment]
@JHJ-MS hello. Do you have any updates on our issue?

**Comment by gmichaud** [comment]
Thanks for posting this @leybal! I used the "frown" button in Excel to give feedback around this, and also commented on the blog post announcing this feature the day it came out ([link](https://techcommunity.microsoft.com/t5/excel-blog/formula-argument-assistance/bc-p/3774495/highlight/true#M3446)) -- others provided similar feedback as well. So far, I haven't seen any response from Microsoft.

**Comment by wh1t3cAt1k** [comment]
@JinghuiMS bringing this to your attention...

**Comment by leybal** [comment]
@JinghuiMS @penglongzhaochina could you share any updates with us?

**Comment by leybal** [comment]
@gmichaud thanks for adding the similar issue. We will be waiting for updates. Maybe you have any timeline updates?

**Comment by gmichaud** [comment]
@leybal have not heard back anything regarding ScreenTips and custom functions unfortunately :(

**Comment by ruijiaMS** [comment]
Hi @leybal and @gmichaud, thanks for letting us know this issue! We confirm this issue is a bug for custom function in Excel Online and have raised a new internal bug #8420025 to track the progress. Any new update will post here!

**Comment by wh1t3cAt1k** [comment]
Hi @ruijiaMS, has there been a development on this item? 

Right now complex custom functions with multiple arguments are nearly unusable in Excel Online because of this bug and #3916 (neither the Insert Function nor the tooltip work very well), and our support team and customers are complaining about this.

**Comment by ruijiaMS** [comment]
Hi @wh1t3cAt1k, 
the custom function arguments missing fix has been deployed to Excel Online Production, I just have a try on it and looks like the local cf functions can be shown correctly for both edge and chrome. 
Please have a try on your stations, thanks!
![image](https://github.com/OfficeDev/office-js/assets/113165331/5a713187-8438-4bdf-a6a8-839955e3c300)

**Comment by wh1t3cAt1k** [comment]
@ruijiaMS nothing has changed. The function itself is being shown, as was indicated in the original bug description. However, its arguments are not.

For an internal function:

![image](https://github.com/OfficeDev/office-js/assets/5159662/ca412715-0be3-42b9-b958-1cf217808a66)

For [Velixo](https://appsource.microsoft.com/en-us/product/office/wa200002311?tab=overview)'s GI() custom function the **function name** is displayed:

![image](https://github.com/OfficeDev/office-js/assets/5159662/32547f78-e345-4b6f-9122-0c33cb0c4449)

But as soon as I open the bracket I don't see any arguments:

![image](https://github.com/OfficeDev/office-js/assets/5159662/18ac425b-ed93-40c4-ab15-b0a4d6a74ee4)

Unfortunately it is not very useful if we can just see the function name, because our custom functions contain complex arguments, which our users need hints about.

**Comment by ruijiaMS** [comment]
Hi @wh1t3cAt1k,
Thanks for your details here! we have released the function name display issue on production as you are seeing, but for the argument missing and dialog issue mentioned in #3916, our team have raised tasks 8668416 and 8668388 to refactor the argument assistance and improve authoring experience.
We keep very high priority to deal with and release these tasks, and any further update will post here.

**Comment by wh1t3cAt1k** [comment]
Thank you @ruijiaMS, hoping for an update from you soon here.

**Comment by wh1t3cAt1k** [comment]
@ruijiaMS could you please let us know if this has been addressed and what would be the expected build number with the fix? Thank you!

**Comment by ruijiaMS** [comment]
@wh1t3cAt1k Of course, every work item is in progress, we could not give an expected build number now but we will post a number as it is available.

**Comment by wh1t3cAt1k** [comment]
@ruijiaMS @adrianwu8516 just following up on whether 8668416 / 8668388 are still treated with high priority on your side?

I just tried on Excel Online and it seems that even though the function itself is shown in the drop-down:

![image](https://github.com/OfficeDev/office-js/assets/5159662/9e06f19b-755f-4262-85f5-f424b1c8b726)

When I open the bracket, there is no inline tooltip for its arguments:

![image](https://github.com/OfficeDev/office-js/assets/5159662/710f04b8-4e00-4e4e-b645-9fb166b96f98)

I would expect to see a similar experience to built-in functions like SUM:

![image](https://github.com/OfficeDev/office-js/assets/5159662/8633bd4c-2732-4b92-91d5-d97d8cb61229)

**Comment by ruijiaMS** [comment]
Hi @wh1t3cAt1k ,
Sorry for the delay here! This work is still in progress and I will update this issue as soon as it is done.

**Comment by Tom-H2oThermo** [comment]
Please note that the screentips is only a part of the issue, please refer to #4431. When you start entering your formula, clicking on the fx to the left of the formula bar in Excel Desktop, you get something like this:
![image](https://github.com/OfficeDev/office-js/assets/25697066/aefed85d-014c-4982-aef3-1f8f75140a1b)
Whereas clicking on the fx in the web version of Excel you get Insert Function, which isn't very useful:
![image](https://github.com/OfficeDev/office-js/assets/25697066/e370961a-563a-4923-99a4-2ca1a5528c86)

**Comment by ruijiaMS** [comment]
@Tom-H2oThermo,
Thanks for your feedback! I will forward your concern back and track it with high priority! and we will have a complete set of solutions to this problem, including but not limited to the problems mentioned above.

**Comment by Tom-H2oThermo** [comment]
Thank you sir!

**Comment by wh1t3cAt1k** [comment]
Testing on Excel Online **16.0.18207.42311** with the [Velixo add-in](https://appsource.microsoft.com/en-us/product/office/wa200002311?tab=overview) the problem still seems to reproduce, no argument tooltips are being shown:

<img width="418" alt="image" src="https://github.com/user-attachments/assets/dbd4752a-eb06-4f09-8134-dc8196ea133e">

**Comment by rajatavams** [comment]
Hello All,

The Fix was released. Kindly let us know if you are still facing the same issue.

Thanks!

**Desktop**

<img width="908" height="135" alt="Image" src="https://github.com/user-attachments/assets/be757dbe-2ad2-49f1-88d7-8168503233d4" />

**Web**

<img width="341" height="289" alt="Image" src="https://github.com/user-attachments/assets/61021bfb-9e3a-48a3-8fde-f46b628206c7" />
